### PR TITLE
support give permission to car

### PR DESCRIPTION
**Pull request by chaooe**
https://github.com/appium/appium/issues/14076

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -370,7 +370,7 @@ const getBuildToolsDirs = _.memoize(async function getBuildToolsDirs (sdkRoot) {
 const extractMatchingPermissions = function (dumpsysOutput, groupNames, grantedState = null) {
   const groupPatternByName = (groupName) => new RegExp(`^(\\s*${_.escapeRegExp(groupName)} permissions:[\\s\\S]+)`, 'm');
   const indentPattern = /\S|$/;
-  const permissionNamePattern = /android\.permission\.\w+/;
+  const permissionNamePattern = /android\.\w*\.permission\.\w+/;
```
Comment (mykola-mokhnach) [inline lib/helpers.js]: Please change the regex to 
`/android\.\w*\.?permission\.\w+/`

**Comment by mykola-mokhnach** [comment]
Could you please sign the CLA and add at least one unit test for this helper. The test could be added to https://github.com/appium/appium-adb/blob/master/test/unit/helper-specs.js

**Comment by chaooe** [comment]
Yes. I have signed the CLA. and try to add the unit test.

**Comment by mykola-mokhnach** [comment]
The CLA bot wants that the email address used to sign the licence is present in local settings of the github account, which creates the PR. Please make sure the local account settings are correct, so the bot could verify the signing fact.

File: test/unit/helper-specs.js [hunk 2]
```diff
@@ -422,4 +422,67 @@ describe('helpers', withMocks({fs}, function (mocks) {
       should.throw(() => parseJsonData(broadcastOutput, ''));
     });
   });
+
+  describe('parsePermissions', function () {
```
Comment (mykola-mokhnach) [inline test/unit/helper-specs.js]: 👍

**Comment by chaooe** [comment]
Ok. Let me fix them

File: test/unit/helper-specs.js [hunk 3]
```diff
@@ -433,44 +433,30 @@ describe('helpers', withMocks({fs}, function (mocks) {
...
       android.permission.WRITE_EXTERNAL_STORAGE
       android.permission.READ_EXTERNAL_STORAGE
     install permissions:
       android.car.permission.CAR_VENDOR_EXTENSION: granted=true
-      android.car.permission.CAR_CONTROL_AUDIO_VOLUME: granted=true
-      android.permission.BLUETOOTH: granted=true
-      android.permission.INTERNET: granted=true
+      android.123.permission.CAR_CONTROL_AUDIO_VOLUME: granted=true
+      android.xxx.BLUETOOTH: granted=true
+      com.google.android.c2dm.permission.RECEIVE: granted=true
```
Comment (mykola-mokhnach) [inline test/unit/helper-specs.js]: 😍

**Pull request merged**